scripts/firepit-builder/pipeline.js

```diff
@@ -33,25 +33,25 @@ cd("firepit_pipeline");
 const workdir = pwd();
 
 npm("init", "-y");
-npm("install", firebaseToolsPackage);
+
+if (fs.existsSync(firebaseToolsPackage)) {
```

Comment:
Seems like `firebaseToolsPackage` can be something like `firebase-tools@latest` which does not make any sense as an argument to `fs.existsSync()` right?

Reply:
That's exactly the point, if it's an npm package like `firebase-tools@latest` then the step to manually pack the module isn't needed, so this fails and it skips this.

Reply:
SG